### DISCUSSION TITLE
How can i make my own build for all OS of kolibri

### DISCUSSION THREAD

**Discussion by dbhattkenpath** [learningequality/kolibri#9703] [Q&A] [answered]
Hi Team,

I want to know that what are the steps to make my own kolibri package for windows(.exe), for linux(.deb), for mac(.dmg).

**Comment by radinamatic** [comment, marked as answer]
@dbhattkenpath Installers of Kolibri for various OSes are available on our [Download page](https://learningequality.org/download/). You can also check the repositories for [Windows](https://github.com/learningequality/kolibri-installer-windows), [Linux](https://github.com/learningequality/kolibri-installer-debian) and [macOS](https://github.com/learningequality/kolibri-app) installers here on Github.

If you are making some kind of modified version of Kolibri on your own, you should be able to search and find resources online on how to package the executable installers for various operating systems, as that is out of the scope of support we offer in this repository.

**Reply by dbhattkenpath** [reply]
Thanks @radinamatic  for this information.
